pkg/apiserver/apiserver.go

```diff
@@ -63,7 +63,7 @@ var (
 			// Use buckets ranging from 125 ms to 8 seconds.
```

Comment:
Please update the help text to include the new field (and remove handler if you stick with removing it)